packages/react/README.md

```diff
@@ -180,76 +179,42 @@ export const ServerComponent = async () => {
 };
 ```
 
-#### Server and Client (experimental)
```

Comment:
I think we should keep this section as is. If we want a simplified React usage it should be an example of using the OpenFeature React SDK under "openfeature-web-provider" but not important to add in this PR.